src/components/InputSwitch/index.tsx

```diff
@@ -100,6 +102,7 @@ const InputSwitch = (props: InputSwitchProps): JSX.Element => {
                             style={{ margin: "0 6px" }}
                             value={numericValue}
                             onChange={handleInputChange}
+                            type="number"
```

Comment:
```suggestion
                            type="number"
                            tooltip={{ formatter: (value) => value?.toFixed(2) }}
```

Reply:
I think the root of the issue here is that we're rounding off the values, but not the maxValues, so we're getting an issue where we're trying to allow the user to input the max of 0.16199999999, but then we round that to 0.162 which is above the max.

I'll add the same rounding to the maxValue and that will solve this issue!